docs/clear-containers-3.0-quick-install.md

```diff
@@ -0,0 +1,158 @@
+```
+
+### Shim
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/shim
+$ ./autogen.sh
+$ make
+(CDPATH="${ZSH_VERSION+.}:" && cd . && /bin/sh /home/foo/go/shim/missing autoheader)
+rm -f stamp-h1
+touch config.h.in
+cd . && /bin/sh ./config.status config.h
+config.status: creating config.h
+config.status: config.h is unchanged
+make  all-am
+make[1]: Entering directory '/home/foo/go/shim'
+  CC       src/cc_shim-shim.o
+  CC       src/cc_shim-utils.o
+  CC       src/cc_shim-log.o
+  CCLD     cc-shim
+$ sudo make install
+make[1]: Entering directory '/home/foo/go/shim'
+ /usr/bin/mkdir -p '/usr/libexec/clear-containers'
+  /bin/sh ./libtool   --mode=install /usr/bin/install -c cc-shim '/usr/libexec/clear-containers'
+libtool: install: /usr/bin/install -c cc-shim /usr/libexec/clear-containers/cc-shim
+make[1]: Nothing to be done for 'install-data-am'.
+make[1]: Leaving directory '/home/foo/go/shim'
+```
+
+### Runtime
+
+```bash
+$ cd $GOPATH/src/github.com/clearcontainers/runtime
+$ make
+go build -i -ldflags "-X main.commit="ce08f6bc8b73422e94fd9b9450633799680fbfe0" -X main.version=3.0.0-alpha.0" -o cc-runtime .
+$ sudo make install
+install -D cc-runtime /usr/local/bin/cc-runtime
+install -D config/configuration.toml /usr/local/etc/clear-containers/configuration.toml
+```
+
+### Virtcontainers (pause binary only)
+
+This step will only install the **pause** binary included in [https://github.com/containers/virtcontainers](https://github.com/containers/virtcontainers)
+
+```bash
+$ sudo yum install -y glibc-static
+$ cd $GOPATH/src/github.com/clearcontainers/runtime/.ci/
+$ ./install_virtcontainers.sh
+Retrieve virtcontainers repository
+~/go/src/github.com/containers/virtcontainers ~/go/src/github.com/clearcontainers/runtime
+Build pause binary
+make -C pause
+make[1]: Entering directory '/home/foo/go/src/github.com/containers/virtcontainers/pause'
+cc -Os -Wall -Wextra -static   -c -o pause.o pause.c
+cc -o pause pause.o -Os -Wall -Wextra -static
+strip pause
+make[1]: Leaving directory '/home/foo/go/src/github.com/containers/virtcontainers/pause'
+Create /var/lib/clear-containers/runtime/bundles/pause_bundle/bin
+Install pause binary
+~/go/src/github.com/clearcontainers/runtime
+```
+
+## Configure
+
+ Create a link for ```/usr/local/etc/clear-containers/configuration.toml```
+
+```bash
+$ mkdir -p /etc/clear-containers
+$ ln -s /usr/local/etc/clear-containers/configuration.toml /etc/clear-containers/configuration.toml
+```
+
+Edit the file according to your needs.
+
+### Edit Docker systemd service file
+
+```bash
+$ sudo mkdir -p /etc/systemd/system/docker.service.d/
+$ cat << EOF | sudo tee /etc/systemd/system/docker.service.d/clr-containers.conf
+[Service]
+ExecStart=
+ExecStart=/usr/bin/dockerd -D --add-runtime cor=/usr/local/bin/cc-runtime --default-runtime=runc
+
+[Service]
+# Allow maximum number of containers to run.
+TasksMax=infinity
+
+EOF
+```
+
+## Restart the Docker and Clear Containers systemd services
```

Comment:
Please, do not think of the headings as numbered steps. The heading here should read something along the lines: Restarting the system